app/main.py

```diff
@@ -2,5 +2,15 @@
 
 
 def cache(func: Callable) -> Callable:
-    # Write your code here
-    pass
+    runs = {}
+
+    def wrapper(*args, **kwargs) -> int:
+        if args in runs:
```

Comment:
You should check both `args` and `kwargs` because the function might accept the same `args` but different `kwargs`. Additionally, I recommend checking the function name since there could be two different functions that accept the same arguments.